### PR TITLE
Adding an email service

### PR DESCRIPTION
**Pull request by WilliamWood01**

### CODE REVIEW

File: src/modules/users/users.route.ts [hunk 1]
```diff
@@ -140,4 +143,15 @@ router.get(
...
+// router.post('/send', async (req, res) => {
+//   try {
+//     //await mailService.sendOrderMail('willwood98@outlook.com', req.body);
+//     await mailService.sendPrescriptionMail('willwood98@outlook.com', req.body);
+//     res.status(200).json();
+//   } catch (error: any) {
+//     console.error(error);
+//     res.status(500).json({ error: error.message });
+//   }
+// });
```
Comment (adelinavisan97) [inline src/modules/users/users.route.ts:155]: are you going to call this endpoint from the front end? or use the function in the users.service in the createOrder/Prescription functions?

Comment (adelinavisan97) [reply src/modules/users/users.route.ts:155]: the sendOrderMail and sendPrescriptionMail functions I mean

Comment (WilliamWood01) [reply src/modules/users/users.route.ts:155]: nah this was just so I could test it

Comment (WilliamWood01) [reply src/modules/users/users.route.ts:155]: I will be calling the functions directly from the user service when the order or prescription creation endpoints are triggered

Comment (adelinavisan97) [reply src/modules/users/users.route.ts:155]: okay yeah great, looks good! will approve

**Review by adelinavisan97 — APPROVED** [review]

**Pull request merged**